include/solvers/copy_data.h

```diff
@@ -0,0 +1,187 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 - 2021 by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+
+#include <deal.II/dofs/dof_handler.h>
+
+#include <deal.II/lac/full_matrix.h>
+#include <deal.II/lac/vector.h>
+
+#include <vector>
+
+#ifndef copy_data_navier_stokes_h
+#  define copy_data_navier_stokes_h
+
+using namespace dealii;
+
+/**
+ * @brief CopyData class is responsible for
+ * storing the information calculated using the assembly of regular (meaning
+ * non-stablized) equations. This class is used to
+ * initialize, zero (reset) and store the cell_matrix, the cell_rhs and the
+ * dof indices associated with the dofs of the cell.
+ **/
+
+class CopyData
+{
+public:
+  /**
+   * @brief Constructor. Allocates the memory for the cell_matrix and cell_rhs
+   *
+   * @param n_dofs Number of degrees of freedom per cell in the problem
+   */
+  CopyData(const unsigned int n_dofs)
+    : local_matrix(n_dofs, n_dofs)
+    , local_rhs(n_dofs)
+    , local_dof_indices(n_dofs){};
+
+  /**
+   * @brief Resets the cell_matrix and the cell_rhs to zero
+   */
+  void
+  zero()
```

Comment:
Could we change the name of these functions to `reset()` or `set_to_zero()`? I find it more intuitive than just `zero()`

Reply:
Absolutely.

Reply:
Done, will show in next push.